hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/routes.json

```diff
@@ -28,5 +28,44 @@
   "/heatmap/readaccess?startDate=*&path=*&entityType=bucket": "/bucketHeatmap",
   "/heatmap/readaccess?startDate=*&path=*&entityType=key": "/keyHeatmap",
   "/heatmap/readaccess?startDate=*&path=*&entityType=volume": "/heatmap",
-  "/features/disabledFeatures": "/disabledFeatures"
+  "/features/disabledFeatures": "/disabledFeatures",
+  "/keys/open?limit=0": "/keysOpenSummary",
+  "/keys/deletePending?limit=1": "/keysdeletePendingSummary",
+
+  "/containers/mismatch?limit=*&prevKey=11&missingIn=OM" : "/omMismatch1",
+  "/containers/mismatch?limit=*&prevKey=21&missingIn=OM" : "/omMismatch2",
+  "/containers/mismatch?limit=*&prevKey=31&missingIn=OM" : "/omMismatch3",
+  "/containers/mismatch?limit=*&prevKey=41&missingIn=OM" : "/omMismatch4",
+
+  "/containers/mismatch?limit=*&prevKey=*&missingIn=OM" : "/omMismatch",
```

Comment:
in this uri, /omMistmatch <-- this is test uri or normal uri? how about omMismatch1, omMistmatch2, ...

Reply:
@sumitagrawl 
It is test uri to check in local with 3000 port as we are using pagination depending uponlast key so check functionality so mapped to route.json for test data.